docs/src/main/asciidoc/_developer-guide/How_do_I_migrate_my_application_from_AS7_to_WildFly.adoc

```diff
@@ -799,7 +799,7 @@ JAX-RPC Web Services may be identified by the presence of the XML descriptor nam
 </webservices>
 ----
 
-Applications using JAX-RPC should be migrated to use JAX-WS, the current Jakarta EE standard web service framework.
+Applications using Jakarta XML RPC should be migrated to use JAX-WS, the current Jakarta EE standard web service framework.
 
 [[jax-rs-2.0]]
```

Comment:
JAX-RS appears six more times. Of those, JAX-RS 1.X is appropriate.

Also, "JSR 339" and the outdated URL https://jcp.org/en/jsr/detail?id=339 appear.